Sources/Swift/Core/Integrations/Integrations.swift

```diff
@@ -34,12 +34,16 @@ private struct AnyIntegration {
 @_spi(Private) @objc public final class SentrySwiftIntegrationInstaller: NSObject {
     @objc public class func install(with options: Options) {
         let dependencies = SentryDependencyContainer.sharedInstance()
-        let commonIntegrations: [AnyIntegration] = [.init(SwiftAsyncIntegration.self)]
+
+        var integrations: [AnyIntegration] = [.init(SwiftAsyncIntegration.self)]
         #if os(iOS) && !SENTRY_NO_UIKIT
-        let integrations: [AnyIntegration] = commonIntegrations + [.init(UserFeedbackIntegration<SentryDependencyContainer>.self)]
-        #else
-        let integrations: [AnyIntegration] = commonIntegrations
+        integrations.append(.init(UserFeedbackIntegration<SentryDependencyContainer>.self))
+        #endif
+        
+        #if ((os(iOS) || os(tvOS) || (swift(>=5.9) && os(visionOS))) && !SENTRY_NO_UIKIT) || ((os(macOS) || targetEnvironment(macCatalyst)) && !SENTRY_NO_UIKIT)
+        integrations.append(.init(FlushLogsIntegration<SentryDependencyContainer>.self))
```

Comment:
`l`: What's your reason for refactoring this logic to not use "commonIntegrations" and added ones?